bundle/manifests/cc-operator.clusterserviceversion.yaml

```diff
@@ -215,11 +215,7 @@ metadata:
                   "name": "kata-clh",
                   "snapshotter": "nydus"
                 },
-                {
```

Comment:
just one comment for the records (maybe I should document it). We have been updating manually the bundle/manifests files but in reality they should be generated by `make bundle`.

On the case of this PR, first edit the configuration file:
```
diff --git a/config/samples/ccruntime/default/kustomization.yaml b/config/samples/ccruntime/default/kustomization.yaml
index 03c49f9..b22bc16 100644
--- a/config/samples/ccruntime/default/kustomization.yaml
+++ b/config/samples/ccruntime/default/kustomization.yaml
@@ -20,8 +20,6 @@ patches:
       value:
       - name: "kata-clh"
         snapshotter: "nydus"
-      - name: "kata-clh-tdx"
-        snapshotter: "nydus"
       - name: "kata-qemu"
         snapshotter: "nydus"
       - name: "kata-qemu-tdx"
```

Then run `make bundle IMG=quay.io/confidential-containers/operator:v0.8.0` resulting in:
```
diff --git a/bundle/manifests/cc-operator.clusterserviceversion.yaml b/bundle/manifests/cc-operator.clusterserviceversion.yaml
index bb27a36..3be280d 100644
--- a/bundle/manifests/cc-operator.clusterserviceversion.yaml
+++ b/bundle/manifests/cc-operator.clusterserviceversion.yaml
@@ -215,10 +215,6 @@ metadata:
                   "name": "kata-clh",
                   "snapshotter": "nydus"
                 },
-                {
-                  "name": "kata-clh-tdx",
-                  "snapshotter": "nydus"
-                },
                 {
                   "name": "kata-qemu",
                   "snapshotter": "nydus"
@@ -250,8 +246,8 @@ metadata:
       ]
     capabilities: Basic Install
     categories: Security
-    containerImage: quay.io/confidential-containers/operator:v0.8.0
-    createdAt: "2023-11-15T15:02:03Z"
+    containerImage: quay.io/confidential-containers/operator:latest
+    createdAt: "2023-11-30T13:25:18Z"
     operators.operatorframework.io/builder: operator-sdk-v1.30.0
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
   name: cc-operator.v0.8.0
diff --git a/config/manager/kustomization.yaml b/config/manager/kustomization.yaml
index 1c64f81..b247c45 100644
--- a/config/manager/kustomization.yaml
+++ b/config/manager/kustomization.yaml
@@ -13,4 +13,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: quay.io/confidential-containers/operator
-  newTag: latest
+  newTag: v0.8.0
diff --git a/config/samples/ccruntime/default/kustomization.yaml b/config/samples/ccruntime/default/kustomization.yaml
index 03c49f9..b22bc16 100644
--- a/config/samples/ccruntime/default/kustomization.yaml
+++ b/config/samples/ccruntime/default/kustomization.yaml
@@ -20,8 +20,6 @@ patches:
       value:
       - name: "kata-clh"
         snapshotter: "nydus"
-      - name: "kata-clh-tdx"
-        snapshotter: "nydus"
       - name: "kata-qemu"
         snapshotter: "nydus"
       - name: "kata-qemu-tdx"
```

ps: don't know why it changed `containerImage` to latest, it is supposed to keep v0.8.0, probably a misconfiguration somewhere

Reply:
I've re-gened the bundle using your command, and fixup'd. It looks like it's mostly the timestamp and formatting that have updated. We'll see how the tests go once https://github.com/confidential-containers/operator/pull/299 is merged